src/modules/drawing/components/DrawingTooltip.vue

```diff
@@ -86,28 +86,29 @@ function onPointerMove(event) {
     let featureDrawingMode = featureUnderCursor?.get('type').toUpperCase()
     let translationKeys
 
-    if (hoveringSelectedFeature) {
-        mapElement.classList.add(cssGrab)
-        mapElement.classList.remove(cssPointer)
-
+    if (hoveringSelectedFeature && !drawingMode.value) {
         let hoveringVertex = getVertexCoordinates(featureUnderCursor).some((coordinate) => {
             let pixel = olMap.getPixelFromCoordinate(coordinate)
             return pointWithinTolerance(pixel, event.pixel, DRAWING_HIT_TOLERANCE)
         })
+        translationKeys = `select_feature_${featureDrawingMode}`
 
-        // Display a help tooltip when modifying
-        if (hoveringVertex || pointFeatureTypes.includes(featureDrawingMode)) {
```

Comment:
What was the problem of detecting that it was a point type here?

Reply:
The problem is that even if you don't hover the vertex itself but rather the balloon then the tippy of the vertex shows